extern/boostd-data/yugabyte/migrator.go

```diff
@@ -78,3 +83,19 @@ func (m *Migrator) Migrate(ctx context.Context) error {
 
 	return nil
 }
+
+// StripLoadBalance is used as a workaround to pass YGB-PGX formatted string to SQL formatted string
+func StripLoadBalance(connectString string) (string, error) {
```

Comment:
Have we tested it against a multi-node deployment? If you run a migration, would it update all nodes?

Reply:
This is internal DB migration and not the migration to LID.

Reply:
Ok, so there is always one node right?